test/parallel/test-buffer-regression-649.js

```diff
@@ -6,8 +6,13 @@ const SlowBuffer = require('buffer').SlowBuffer;
 
 // Regression test for https://github.com/nodejs/node/issues/649.
 const len = 1422561062959;
-assert.throws(() => Buffer(len).toString('utf8'));
-assert.throws(() => SlowBuffer(len).toString('utf8'));
-assert.throws(() => Buffer.alloc(len).toString('utf8'));
-assert.throws(() => Buffer.allocUnsafe(len).toString('utf8'));
-assert.throws(() => Buffer.allocUnsafeSlow(len).toString('utf8'));
+assert.throws(() => Buffer(len).toString('utf8'),
+              /Invalid typed array length/);
```

Comment:
If you check the entire error message, you can add `^` and `$` to the regular expression. Also, since the regex is the same for all the checks, you could move it to a variable.